src/ensembl/src/shared/components/view-in-app/ViewInApp.scss

```diff
@@ -0,0 +1,22 @@
+@import 'src/styles/common';
+
+.viewIn {
+  color: $grey;
+  width: 50px;
+}
+.entityViewerAppLinkButtons {
```

Comment:
If ViewInApp is a generic component, why do its classes have "entityViewer" in their names?

Reply:
True. Thanks